docs/DeveloperGuide.md

```diff
@@ -406,9 +406,9 @@ file with the modified workout list whenever a workout is deleted in the applica
 <br><br>
 An alternative considered was to find the workout to be deleted in the resource file, and then
 remove that workout. This is a faster implementation, however, more complicated due to the way
-the workouts are formatted and stored in the `workouts.txt` file.
+the workout data are formatted and stored in the `workouts.txt` file.
```

Comment:
Perhaps the following sentence could be rephrased:
| Original | Revised |
| --- | --- |
| This is a faster implementation, however, more complicated due to the way the workout data are formatted and stored in the `workouts.txt` file. | **While** this is a ~faster~ **more efficicient** implementation, ~however, more complicated~ **it is more complex** due to the way the workout data are formatted and stored in the `workouts.txt` file. |

Note: I bolded the text to highlight the edited words, but they should not be bolded in the actual change 😬

Reply:
Thanks!